src/app/portfolio/portfolio.component.html

```diff
@@ -40,26 +40,29 @@
                      src="{{ item.campaign.photo | toUrlIPFS }}" alt="Project image">
               </a>
             </div>
-            <div class="w-full bg-white rounded-b-3xl lg:rounded-none lg:rounded-r-3xl p-6">
-              <div class="flex">
-                <div class="bg-gray-200 px-8 rounded-full py-2 flex items-center">
-                  <span class="font-bold">{{ item.tokenValue | formatUnit:'bignumToWei' | currencyDefault }}</span>
-                  <span class="ml-2">Invested</span>
+            <div class="flex flex-col justify-between w-full bg-white rounded-b-3xl lg:rounded-none lg:rounded-r-3xl
+                        p-6">
+              <div>
+                <div class="flex">
+                  <div class="bg-gray-200 px-8 rounded-full py-2 flex items-center">
+                    <span class="font-bold">{{ item.tokenValue | formatUnit:'bignumToWei' | currencyDefault }}</span>
+                    <span class="ml-2">Invested</span>
+                  </div>
                 </div>
-              </div>
-              <div class="text-3xl mt-4 lg:text-4xl tracking-wider font-light uppercase">
+                <div class="text-3xl mt-4 lg:text-4xl tracking-wider font-light uppercase">
                 <a routerLink="{{ '/offers/' + item.campaign.ansName | issuerPath }}">
                   {{ item.campaign.name }}
                 </a>
               </div>
+              </div>
               <div class="mt-2 flex flex-col lg:flex-row justify-start lg:justify-between">
                 <div class="w-full lg:mr-16">
                   <app-funding-progress [campaign]="item.campaign"></app-funding-progress>
                 </div>
                 <div class="flex items-end">
                   <button *ngIf="!item.campaign.finalized" app-action-button
-                          class="w-full lg:w-52 lg:px-20 lg:ml-8 mt-8 lg:mt-0 py-2 bg-transparent
-                          text-black font-semibold rounded-full whitespace-nowrap"
+                          class="w-full lg:w-52 lg:px-20 lg:ml-8 mt-8 lg:mt-0 py-2 bg-red-500
```

Comment:
The "Cancel investment" button is red in the design, it was transparent here.
Before:
![Screenshot 2021-09-22 at 16 09 25](https://user-images.githubusercontent.com/12549625/134380706-24588109-9980-4d21-9b12-49f540df0aa5.png)
After:
![Screenshot 2021-09-22 at 16 09 42](https://user-images.githubusercontent.com/12549625/134380714-5287616d-add5-49b6-b4d3-190358d87900.png)

Reply:
In my opinion, a red button indicates something dangerous and invasive, and cancel investment is not one of these behaviors, that's why Mislav originally set it to this styling. Can you set it back to white/black combo?